### PR TITLE
Amend prophet startup logic in order to improve readability;

### PR DESCRIPTION
**Pull request by cnutshell**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

Fixes #

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Comment by cnutshell** [comment]
Aim to readability improvement:
- Change configuration field `storage-node` to `prophet-node`;
- Amend name for some variables and functions;
- Amend code structure and format;

File: components/prophet/election/election.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ var (
 // Elector a leader elector
 type Elector interface {
 	// CreateLeadship create a leadership
-	CreateLeadship(purpose string, nodeName, nodeValue string, allowCampaign bool, becomeLeader, becomeFollower func(string) bool) *Leadership
+	CreateLeadship(purpose string, nodeName, nodeValue string, isProphet bool, becomeLeaderFunc, becomeFollowerFunc func(string) bool) *Leadership
```
Comment (zhangxu19830126) [inline components/prophet/election/election.go:32]: Elector is a relatively independent module that can be used in isolation from prophet. Suggest that allowCampaign be renamed allowBecomeLeader

**Review by zhangxu19830126 — CHANGES_REQUESTED** [review]

File: components/prophet/prophet.go [hunk 2]
```diff
@@ -119,74 +119,89 @@ func NewProphet(cfg *config.Config) Prophet {
...
 	}
 
-	elector, err = election.NewElector(etcdClient,
+	// elector is mainly a wrapper of etcd client
+	elector, err = election.NewElector(
+		etcdClient,
 		election.WithLeaderLeaseSeconds(cfg.Prophet.LeaderLease),
 		election.WithEmbedEtcd(etcd),
-		election.WithLogger(log.Adjust(cfg.Logger).Named("elector")))
+		election.WithLogger(log.Adjust(cfg.Logger).Named("elector")),
```
Comment (zhangxu19830126) [inline components/prophet/prophet.go:146]: use logger.Named("elector")

**Review by zhangxu19830126 — CHANGES_REQUESTED** [review]

**Review by zhangxu19830126 — APPROVED** [review]
LGTM

**Pull request merged**